### PR TITLE
fix: image-cloze-association create schemas for PD-1095

### PR DESCRIPTION
**Pull request by PatriciaRomaniuc**
https://illuminate.atlassian.net/browse/PD-1095
https://illuminate.atlassian.net/browse/PD-1114

### CODE REVIEW

File: packages/pie-models/src/pie/image-cloze-association/index.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    rationaleEnabled?: boolean;
+
+    /** Indicates if Teacher Instructions are enabled */
+    teacherInstructionsEnabled?: boolean;
+
+    /** Indicates if Student Instructions are enabled */
+    studentInstructionsEnabled?: boolean;
+
+    /** The image over which the responses will be dragged*/
+    image?: {
```
Comment (andreeapescar) [inline packages/pie-models/src/pie/image-cloze-association/index.ts:61]: I guess we could create a new Interface for this object?

File: packages/image-cloze-association/controller/src/__tests__/index.test.js [hunk 2]
```diff
@@ -323,8 +327,8 @@ describe('controller', () => {
           validResponse: {
             score: 1,
             value: [
-              [rhomb, square],
-              [rhomb, square, trapeze],
+              {images: [rhomb, square]},
```
Comment (andreeapescar) [inline packages/image-cloze-association/controller/src/__tests__/index.test.js:330]: Let's add tests where we have {}, or { images: null} etc. (This will require to add sanity checks on the next file).

**Review by andreeapescar — CHANGES_REQUESTED** [review]
Also, please make sure you commit with **BREAKING CHANGE:**  .

**Review by andreeapescar — APPROVED** [review]

**Comment by andreeapescar** [comment]
@PatriciaRomaniuc I'm afraid we
